judo-runtime-core-jsl-itest/models/TransferAggregationModel/src/test/java/hu/blackbelt/judo/runtime/core/jsl/transfer/MappedTransferAssociationAggregationTest.java

```diff
@@ -375,6 +381,7 @@ public void testSingleRequiredAssociationAggregationRelationOnTransfer() {
      */
     @Test
     @TestCase("MultiAssociationAggregationRelationOnTransfer")
+    @Disabled
```

Comment:
Is this intentional? If so, maybe we should add a ticket as note?

Reply:
Please give ticket number for disabled test.

Reply:
[210fa36](https://github.com/BlackBeltTechnology/judo-runtime-core-jsl/pull/244/commits/210fa3623369ae45e685aeb21cee5392ea0181c6)